starboard/shared/starboard/player/filter/testing/adaptive_audio_decoder_test.cc

```diff
@@ -382,7 +382,7 @@ vector<vector<const char*>> GetSupportedTests() {
   }
 
   vector<const char*> supported_files =
-      GetSupportedAudioTestFiles(kExcludeHeaac, 6, "audiopassthrough=false");
```

Comment:
The new default behavior will always have "audiopassthrough=true", but the old code here setting it to false. Can you check if the tests which used to have "audiopassthrough=false" can still work as expected?